frontend/src/i18n/locales/en.yaml

```diff
@@ -487,13 +487,17 @@ manage:
         series:
           heading: Series
           invalid: Please select a series
+          findable-series-note: >
+            You can only select series here which are listed or which you have write-access to.
         metadata:
           heading: Metadata
 
       event:
         event:
```

Comment:
I don't think this nesting makes a lot of sense, but again, nothing to change in this PR. Not even introduced here, I just noticed it while reviewing.

Reply:
Me and you both, I'm always kinda sad when I see this super deep nesting in the realm edit tools but... wasn't me and I don't want to touch it necessarily :D